ExchangeSharp/API/Exchanges/BL3P/ExchangeBL3PAPI.cs

```diff
@@ -270,6 +277,22 @@ protected override async Task<ExchangeOrderResult> OnPlaceOrderAsync(ExchangeOrd
 			return orderDetails;
 		}
 
+		protected override async Task<ExchangeOrderBook> OnGetOrderBookAsync(string marketSymbol, int maxCount = 100)
+		{
+			if (string.IsNullOrWhiteSpace(marketSymbol))
+				throw new ArgumentException("Value cannot be null or whitespace.", nameof(marketSymbol));
+
+			var resultBody = await MakeRequestAsync($"/{marketSymbol}/money/depth/full")
+				.ConfigureAwait(false);
+
```

Comment:
Not necessary, the On* methods are wrapped by sync context remover

Reply:
Nope, I didn't realize that here it would not have a sync context. I usually prefer using `ConfigureAwait(false)` because of the clarity that explicit code provides, even though it can be repetitive.
Anyway, I will remove it.

Reply:
Done.